TS03-Subset-Data-and-No-Data-Values-In-R.Rmd

```diff
@@ -39,66 +41,58 @@ After completing this activity, you will:
+of setting the working directory in `R` can be found here.]({{site.baseurl}}/R/Set-Working-Directory "R Working Directory Lesson") 
+lesson prior to beginning this lesson.
 
-* [ Brief Time Series in R - Simple Plots with qplot & as.Date Conversion]({{ site.baseurl}}/R/Brief-Tabular-Time-Series-qplot/)
-* [Understanding Time Series Metadata]({{ site.baseurl}}/R/Time-Series-Metadata/)
-* [Convert Date & Time Data from Character Class to Date-Time Class (POSIX) in R]({{ site.baseurl}}R/Time-Series-Convert-Date-Time-Class-POSIX/)
-* [Refining Time Series Data - NoData Values and Subsetting Data by Date in R]({{ site.baseurl}}/R/Subset-Data-and-No-Data-Values/)
-* [Subset and Manipulate Time Series Data with dplyr]({{ site.baseurl}}/R/Time-Series-Subset-dplyr/)
-* [Plotting Time Series with ggplot in R]({{ site.baseurl}}/R/Time-Series-Plot-ggplot/)
-* [Plot uisng Facets and Plot Time Sereis with NDVI data]({{ site.baseurl}}/R/Time-Series-Plot-Facets-NDVI/)
-* [Converting to Julian Day]({{ site.baseurl}}/R/julian-day-conversion/)
+**Challenge Code:** NEON Data lesson often contain challenges that reinforce 
+learned skills. If available, the code for challenge solutions is found in a 
+downloadable `R` script available on the footer of each lesson page.
 
-###Sources of Additional Information
+###Additional Resources
 
 </div>
 
-#The Data Approach
-We'll learn now how to subset data and how to find and deal with missing data 
-values, while we continue to explore the relationship between our variables of
-interest, air temperature, precipitation, and photosynthetically active
-radiation (PAR) in regards to plant phenology.  
+#Cleaning Time Series Data 
+No dataset is perfect. It is common to encounter, large files containing more
+data than we need for our analysis. It is also common to encounter `noData` values
+that we need to account for when analyzing our data. 
+
+In this tutorial, we'll learn about ways to both manage `noData` values and also 
+subset and export a portion of an R object as a new `.CSV` file. 
+
+In this lesson, we will work with atmospheric data, containing air temperature, 
+precipitation, and photosynthetically active radiation (PAR) data  - metrics that 
+are key drivers of phenology. Our study area is the NEON Harvard Forest Field Site.
+
+We will use the `lubridate` and `ggplot2` packages. Let's load those first.
 
-Prior to starting we need to load the appropriate libraries (`lubridate` and
-`ggplot2`).  If you have not already done so, we must also load the .csv file for
-15-minute micrometeorological data ( file: `hf001-10-15min-m.csv`) and convert 
-the `datetime` column to a POSIXct class date-time data type. 
+If you have not already done so, import the `hf001-10-15min-m.csv` file, which contains
+our atmospheric data for Harvard Forest. Convert the `datetime` column to a `POSIXct` 
+class: [As covered in Time Series Lesson 02]({{ site.baseurl }}/R/Time-Series-Metadata/). 
```

Comment:
Everywhere else we have the lesson title as the link, not Time Sereis Lesson #.  Please change to  Dealing With Dates & Times in R - as.Date, POSIXct, POSIXlt